lib/apiservers/engine/backends/container.go

```diff
@@ -349,30 +349,27 @@ func (c *Container) ContainerRename(oldName, newName string) error {
 func (c *Container) ContainerResize(name string, height, width int) error {
 	defer trace.End(trace.Begin(name))
 
-	// save the name provided to us so we can refer to it if we have to return an error
-	paramName := name
```

Comment:
looks like this was intentional and by removing that we are expanding names from short ones to long ones. eg;
docker stop 1 should say 1 is already stopped instead of 1231euayfghhjsegfqw236rgq is already stopped